### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Signing in to Azure Storage_Toggle Explorer Pane - Tree View_Role:Role defined for "Collapse All" and "Refresh All" element is link but is clearly a button.

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/AzureStorageExplorer#5702]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #A11yWCAG2.1; #DesktopApp; #A11ySev2; #BM_Azure Tools Storage Explorer_May2022; #Azure Tools Storage Explorer; #WCAG4.1.2; #FTP;

## Actual results:
Role defined for "Collapse All" and "Refresh All" element is link.

## Expected results:
Role should be defined as Buttons for "Collapse All" and "Refresh All" element.

## Environmental Details:
Application: Microsoft Azure Storage Explore
version: 1.24.0

## OS Details
Microsoft Windows 11 Enterprise
version 21H2 OS Build 22000.613

## Repro Steps:

1.  Launch Storage Explorer
2.  Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Open Explorer" button.
4. Traverse through the tree view opened on left of screen for whatever tasks or subscription selected.
5. Verify Refresh All", "Collapse All" element present in explorer pane is MAS compliant.

## User Impact:
Users will be confused if buttons are announced as links.

## Attachments

![5702_A11y_AzureStorageToolExplorer_CollapseAll RefreshAll](https://user-images.githubusercontent.com/105198219/168732408-72bc5bd2-4c82-41d6-9af4-6949ab15e730.png)

**Comment by JasonYeMSFT** [comment]
We have switched to use real button elements for the "Collapse All" and "Refresh All" buttons.

**Comment by MRayermannMSFT** [comment]
Hi @ManishaC1 

Please verify that this bug is fixed using the Storage Explorer build at: https://storageexplorerpublish.blob.core.windows.net/privatebuilds/20220624.3-accessibility-testing/StorageExplorer-ia32.exe

**Comment by ManishaC1** [comment]
Issue is not reproducing in the provided environment (ver 1.25.0). Hence closing the bug. Please see the below attachment.
![5702_Issue not repro](https://user-images.githubusercontent.com/105198219/175862432-48e089f4-9ec7-43c4-a045-8e8d292f46ed.png)

**Comment by swethakodigi** [comment]
#Closed

**Comment by ManishaC1** [comment]
GithubTags:#Closed;

**Comment by Vishnuvarthancr7** [comment]
GitHubTags:#Closed;
